lib/embed/viewer/was_seed.rb

```diff
@@ -45,6 +45,18 @@ def format_memento_datetime(memento_datetime)
       def external_url
         @purl_object.ng_xml.xpath('//dc:identifier','dc'=>'http://purl.org/dc/elements/1.1/').try(:text)
       end
+
+      def default_body_height
+        400
+      end
+      
+      def div_dimension
```

Comment:
Could this method name be changed? I don't have an alternative... but `div_dimension` feels a bit unclear to me.